modules/network/arbitrum.ts

```diff
@@ -284,4 +284,40 @@ export const arbitrumNetworkConfig: NetworkConfig = {
             interval: every(5, 'minutes'),
         },
     ],
+    poolsWithGauges: [
```

Comment:
This is a temporary solution to test the implementation until we get these pool lists in an automated way (probably pulling them from the RootGauges contract).

For the sake of testing, I just added this new configuration field to 2 network configs: MAINET and ARBITRUM